ext/featurens/osrelease/osrelease.go

```diff
@@ -32,11 +32,12 @@ var (
 	blocklistFilenames = []string{
 		"etc/alpine-release",
 		"etc/centos-release",
+		"usr/lib/centos-release",
 		"etc/fedora-release",
```

Comment:
I am wondering about fedora, too, since it's upstream.

Reply:
We don't support Fedora in Scanner v2, so I considered removing this, but I decided against it because I didn't want to touch anything unrelated to the goal of this PR